### PR TITLE
[TFLite] Add int16x8 support for ABS operator

### PR DESCRIPTION
**Pull request by Tessil**
Hi,

This PR adds int16x8 support for the ABS operator in TensorFlow Lite.

Thibaut

### CODE REVIEW

File: tensorflow/lite/kernels/elementwise_test.cc [hunk 1]
```diff
@@ -123,7 +148,7 @@ TEST(ElementWise, Log) {
   EXPECT_THAT(m.GetTensorShape(m.output()), ElementsAreArray({1, 1, 4, 1}));
 }
 
-TEST(FloatActivationsOpTest, Abs) {
+TEST(ElementWise, Abs) {
```
Comment (Tessil) [inline tensorflow/lite/kernels/elementwise_test.cc:151]: I renamed `FloatActivationsOpTest` to `ElemWise` for `Abs` and `AbsInt8` for coherence with `AbsInt16`. 

In `acceleration_test_list.cc` the Abs test is mentioned as `ElementWise/Abs` so I suppose the `FloatActivationsOpTest` was a mistake but it'd be good to make sure of it.

Comment (akarmi) [reply tensorflow/lite/kernels/elementwise_test.cc:151]: Similarly, if my understanding is correct, it appears that the change in [this line](https://github.com/tensorflow/tensorflow/pull/43098/files#diff-ffc98f167365cff2837139e9af87b3deR724) is a fix to a small error from an earlier commit.

**Review by daverim — APPROVED** [review]
Looks good to me

**Comment by gbaned** [comment]
@Tessil  Can you please resolve conflicts? Thanks!

**Comment by Tessil** [comment]
@gbaned I resolved the conflict, thanks.

I just realised that I create a `toupstream/16x8_mean_operator` branch for this PR instead of `toupstream/16x8_abs_operator` and I can't change it without creating a new PR. As it's only the branch name, it's probably not too important.

**Comment by gbaned** [comment]
Seems auto-merge is not happening but the changes are merged into master now, so we can close this. Thank you for the PR.

**Pull request closed without merging**